cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go

```diff
@@ -215,6 +215,9 @@ type NodeRegistrationOptions struct {
 	// kubeadm writes at runtime for the kubelet to source. This overrides the generic base-level configuration in the kubelet-config-1.X ConfigMap
 	// Flags have higher priority when parsing. These values are local and specific to the node kubeadm is executing on.
 	KubeletExtraArgs map[string]string `json:"kubeletExtraArgs,omitempty"`
+
+	// IgnorePreflightErrors provides a slice of pre-flight errors to be ignored when the current node is registered.
```

Comment:
It's worth noting here that, although "all" is allowed on the command line, it's not allowed in the config.

Reply:
Thanks for clarifying this. I indeed did not know this requirement.

Reply:
Solved, I think. Will push soon.